### DISCUSSION TITLE
dbeaver not showing up in the microsoft store

### DISCUSSION THREAD

**Discussion by RicardCF84** [dbeaver/dbeaver#19066] [General]
### Description

Hi

Approximately until 2/10/23 the software appears in the store and we could install it without problem.

Today 2/14/2023 we detected that it does not appear in the Microsoft store.

We know if this deployment method is going to be included or discarded.

Regards

### DBeaver Version

ALL

### Operating System

Windows

**Comment by E1izabeth** [comment]
We no longer distribute DBeaver via Microsoft Store.

You can see available options to download here https://dbeaver.io/download/

**Reply by Studio384** [reply]
May I ask what the reason for this was? It was incredibly convenient for it to be just a one-click install...

**Reply by E1izabeth** [reply]
We migrated to JDK 17 recently and it brigs us this issue #18960. We tried to fix it, but unfortunately Microsoft Store started rejecting new DBeaver uploads - it gives very common error without any further information of what exactly is wrong with publishing or how to get things work again. I hope Microsoft Store support will help us, but it may take a long time.

**Reply by webJose** [reply]
Hi!  I'm in the same boat, so I definitely hope that it becomes available through the store at some point in the future.  Many thanks for the explanation.

**Comment by RicardCF84** [comment]
Thanks for your reply

**Comment by E1izabeth** [comment]
DBeaver is available on the Microsoft Store again.

**Comment by webJose** [comment]
Excellent news.  Much appreciated.
